### ISSUE TITLE
Star in centre of view frame

### ISSUE THREAD

**Issue by antonyNethues** [evgenyneu/Cosmos#103]
Please consider submitting the following information (if relevant):

* Library setup method: CocoaPods Swift.
* Version of the library. Example: 15.0.
* Xcode version. Example: 9.0
* OS version. Example: iOS 11.

**Comment by antonyNethues** [comment]
i want to make view width same as super view and then start in centre

**Comment by evgenyneu** [comment]
Hi, you can adjust the size of the cosmos view just changing the size of the stars and separation between them. If you want to center the view you can use the centering autolayout constaint.

**Comment by antonyNethues** [comment]
yes , But i don't want to adjust the size i want the frame width fix(let say iPhone width) and content inside (starts group) centered

**Comment by evgenyneu** [comment]
I see, unfortunately, this is not possible at the moment. It has been discussed in #102 and #29. A pull request will be welcome :).

**Comment by antonyNethues** [comment]
Ok Thanks :)

**Comment by diegogarciar** [comment]
here's my workaround
 ```
let viewA = UIView()
let viewB = UIView()
let horizontalStack = UIStackView(arrangedSubviews: [viewA, cosmosView, viewB])
viewA.widthAnchor.constraint(equalTo: viewB.widthAnchor).isActive = true
```
